src/utils/unicodes.js

```diff
@@ -0,0 +1,99 @@
+export const MAP_CONVERSION = {
```

Comment:
No need for a separate file since `modifySubfields` from the old `marc-record-validate` implementation is not useful as explained above. Also, `MAP_CONVERSION` is only meant for this validator and can be declared in the same file as the validator.